src/main/resources/library/attribute-definitions.json

```diff
@@ -5,145 +5,292 @@
+    "library:RS-PD": {
+      "title": "Future use is limited to pediatric research",
+      "type": "string",
+      "enum": ["Yes", "No"],
+      "consentCode": "RS-PD"
+    },
+    "library:RS-POP": {
+      "title": "Future use is limited to research involving a specific population",
+      "type": "array",
+      "items": { "type": "string" },
+      "consentCode": "RS-POP"
+    },
+    "library:futureUseDate": {
+      "title": "Future use is limited to data generated from samples collected after the following consent form date",
+      "type": "string"
+    },
+    "library:orsp": {
+      "type": "string"
+    }
+  },
+  "consentCodes": {
+    "GRU": "For health/medical/biomedical purposes and other biological research, including the study of population origins or ancestry.",
+    "HMB": "Use of the data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry.",
+    "DS": "Use of the data must be related to a particular disease.",
+    "NCU": "Use of the data is limited to non-commercial use.",
+    "NPU": "Use of the data is limited to not-for-profit organizations.",
+    "NDMS": "Use of the data includes methods development research (e.g., development of software or algorithms) ONLY within the bounds of other specified data use limitations.",
+    "NAGR": "Use of the data includes aggregate level analysis to address research questions (e.g., determine variant frequencies across the general population) ONLY within the bounds of other specified data use limitations.",
+    "NCTRL": "Data can be used as a control set ONLY within the bounds of other specified data use limitations (e.g. only for cancer studies).",
+    "RS-G": "Use of the data is limited to studies of particular gender.",
+    "RS-PD": "Use of the data is limited to pediatric research.",
+    "RS-POP": "Use of the data is limited to studies of particular population."
+  },
+  "display": {
+    "primary": ["library:indication", "library:numSubjects", "library:datatype", "library:dataUseRestriction"],
+    "secondary": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian",
+      "library:datasetDepositor", "library:datasetOwner", "library:institute", "library:projectName",
+      "library:reference", "library:dataFileFormats", "library:technology", "library:profilingProtocol",
+      "library:dataUseBSPSampleCollectionID", "library:studyDesign", "library:cellType", "library:coverage",
+      "library:ethnicity", "library:primaryDiseaseSite", "library:broadInternalResearchProjectID",
+      "library:broadInternalResearchProjectName", "library:broadInternalCohortName",
+      "library:broadInternalSeqProjectNumbers"],
+    "consentCodes": ["GRU", "HMB", "DS", "NCU", "NPU", "NDMS", "NAGR", "NCTRL", "RS-G", "RS-PD", "RS-POP"]
+  },
+  "wizard":
+  [
+    {
+      "title": "Define Data Set attributes",
+      "questions":
+      [
+        {"property": "library:datasetName", "required": true},
```

Comment:
I'm confused by why we're defining property attributes in two places. Wouldn't it be more clear if required were part of the top-level properties? Then the list of questions could just be references to the top level properties instead.